config/webpack.dev.js

```diff
@@ -30,7 +30,7 @@ module.exports = (env, argv) => {
                 ],
             },
             devServer: {
-                open: true,
+                open: env.SUPPRESS_OPEN != undefined ? false : true,
```

Comment:
want to run the dev config build, but then launch a debug browser, and don't want to windows to pop open.